circle.yml

```diff
@@ -512,6 +512,53 @@ jobs:
       - spectest
       - collect_coverage_data
 
+  bindings-rust:
+    docker:
+      - image: rust:1-buster
+    steps:
+      - checkout
+      - run:
+          name: Update environment
+          command: |
+            echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
+            apt -qq update
+            apt -yq -t buster-backports install cmake --no-install-recommends
+            apt -yq install libclang-dev clang cmake --no-install-recommends
+            rustup component add rustfmt
+            rustup update
```

Comment:
Why is update needed?

Reply:
To update any of the dependencies (also those needed by rustfmt). This is mostly a noop if the docker image is up to date (as it should be).

Reply:
This seems wrong to be to having docker image being updated and also updating here. But I will not protest.

Reply:
Removed.